### ISSUE TITLE
Stabilization Layer

### ISSUE THREAD

**Issue by Marcelo5444** [pmh9960/iColoriT#15]
The stabilization layer were it can be found? is the here: https://github.com/pmh9960/iColoriT/blob/8a02415aea6f6cc144650b47f38e58476f4e2a0f/iColoriT_demo/modeling.py#L244

**Comment by YeolJ00** [comment]
Yes. That is the stabilization layer.

**Comment by Marcelo5444** [comment]
Hi, checking the stabilizing layer, in our paper diagram its that features forwarded across the Conv2d, keeep the same dimension however, when running the code I see that the:
nn.Conv2d(192, 512, kernel_size=3, stride=1, padding=1, padding_mode='reflect')

Could you explain that, please
